source/js/components/join/join.scss

```diff
@@ -78,6 +78,9 @@
   }
 
   &.full-page {
+    margin-top: 2rem;
+    margin-bottom: 3rem;
+
```

Comment:
This is fine; The utility classes get confusing the way these components are intertwined